src/IntegratorFactory.php

```diff
@@ -185,7 +185,7 @@ public function createReleaseGroupManifestExecutor(): ReleaseGroupManifestExecut
      */
     public function createGitRepository(?string $pathToRepository = null): GitRepository
     {
-        return new GitRepository($pathToRepository ?? (string)getcwd(), new CliRunner());
+        return new GitRepository($pathToRepository ?? APPLICATION_ROOT_DIR, new CliRunner());
```

Comment:
Are we making sure that the path has a trailing slash somewhere along the line?

Reply:
The const was defined in bin/integrator with a slash along the line